.github/workflows/translations.yaml

```
@@ -35,7 +36,7 @@ jobs:
   download:
     name: Download
     needs: upload
-    if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
+    if: secrets.LOKALISE_TOKEN != null && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch')
```

I guess the same could be applied here then:

```suggestion
    if: github.repository_owner == 'home-assistant' && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch')
```